lib/graphiti/adapters/graphiti_api.rb

```diff
@@ -35,6 +35,8 @@ def handle_remote_error(url, json)
       def build_url(scope)
         url = resource.remote_url
         params = scope[:params].merge(scope.except(:params))
+        params[:page] ||= {}
+        params[:page][:size] ||= 999
```

Comment:
I'd like this to be overridable per-remote resource, but this looks good as a general implementation.

Reply:
What would the override be?

Reply:
Something like max_page_size? Just in case a particular api has changed their default.

Reply:
So what you can currently do is

```
has_many :positions, remote: 'url' do
  params do |hash|
    hash[:page][:size] = 50
  end
end
```

Which would customize on a per-endpoint basis. I can't think of when you'd want something more than this though, or quite where it would go

Reply:
Ahh yes I forgot this was possible here. I’m fine with this as long as you are confident this can successfully transition to the eventual “find_in_batches” follow on work.